pages/common/xml-canonic.md

```diff
@@ -0,0 +1,20 @@
+# xml canonic
+
+> Make XML documents canonical.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Make an XML document canonical, preserving comments (default behavior):
+
+`xml canonic --with-comments {{path/to/input.xml|URI}} >{{path/to/output.xml}}`
+
+- Make an XML document canonical, removing comments:
+
+`xml canonic --without-comments {{path/to/input.xml|URI}} >{{path/to/output.xml}}`
+
+- Exclusive XML canonicalization, using an XPATH from a file, preserving comments:
```

Comment:
Could you maybe rephrase this so that it starts with a verb?

Reply:
You're not going to like the verb. Their manual avoided verbifying an adjective for the same reason.

- Canonicalize exclusively, using an XPATH from a file, preserving comments:

This is less weird:

- Make exclusively canonical, using an XPATH from a file, preserving comments:

Please don't ask me to explain "exclusive". Much reading required.